web/src/layout/StackedLayout.vue

```diff
@@ -1,17 +1,19 @@
 <template>
-    <Navbar v-if="!loading"></Navbar>
```

Comment:
The HTML is not identical if you put it into a div, which is why I used a v-if test on each of the components.

I suggest keeping it as clean as possible (including the resulting HTML)